vector/src/main/java/im/vector/app/features/analytics/impl/DefaultVectorAnalytics.kt

```diff
@@ -123,10 +124,18 @@ class DefaultVectorAnalytics @Inject constructor(
                     Timber.tag(analyticsTag.value).d("User consent updated to $consent")
                     userConsent = consent
                     optOutPostHog()
+                    initOrStopSentry(consent)
                 }
                 .launchIn(globalScope)
     }
 
+    private fun initOrStopSentry(userConsent: Boolean) {
```

Comment:
`userConsent` is already a class member, this is generally not a good idea to shadow class members.
Maybe just remove this parameter (as per `optOutPostHog()`) ?